netman/adapters/shell/base.py

```diff
@@ -25,3 +28,11 @@ def quit(self, command):
 
     def get_current_prompt(self):
         raise NotImplemented()
+
+    @staticmethod
+    def set_default_command_timeout(command_timeout):
```

Comment:
i don't really see a reason for this method why not directly access the property?